docs/source/guides/troubleshooting.rst

```diff
@@ -36,3 +36,19 @@ Problem
   What this usually telling you is that you have a scala version mismatch (Scala classes from 2.10 and 2.11 or 2.11 and 2.12 etc.).
   The best way we have found to resolve these inconsistencies is to do `mvn dependency:tree -Dincludes:scala-library::` and search
   the output for anything containing the unwanted version of Scala. For instance `scalatest_2.10` if you were using ``Scala 2.11``.
+
+
+Swagger
+----------------
+
+I am seeing resources from clients being treated as service resource by swagger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Problem
+  Beadledom clients are automatically generated from an API resources annotated with swagger and JAX-RS annotations.
```

Comment:
1. from ~an~ API resources
2. I put `JAX-RS` before `Swagger` in `... with swagger and JAX-RS annotations`. Also, capitalize swagger.